core/pom.xml

```diff
@@ -45,6 +45,7 @@
                     <artifactSet>
                         <includes>
                             <include>tc.oc.pgm:*:*</include>
+                            <include>it.unimi.dsi:fastutil:*</include>
```

Comment:
Where do we use this? This is a big library, so I'm wondering if there's a way we can do this without it?

Reply:
Maybe we add a utility in `CollectionUtils` (I think that exists?) to create a collection with this, otherwise it fallback to the standard library. Newer server runtimes ship with this and I'd rather not bundle it.